apps/wallet-mobile/src/features/Discover/common/hooks.tsx

```diff
@@ -15,9 +15,9 @@ export const useConnectWalletToWebView = (wallet: YoroiWallet, webViewRef: React
 
   const sendMessageToWebView = (event: string) => (id: string, result: unknown, error?: Error) => {
     if (error) {
-      Logger.info('DappConnector', 'sending error to webview', error, 'as a response to', event)
+      logger.debug('useConnectWalletToWebView: error sending to webview', {error, event})
```

Comment:
```suggestion
      logger.debug('useConnectWalletToWebView: sending error to webview', {error, event})
```

⬆️  because it's sending an error response to webview. Feel free to ignore, maybe it's just my wrong interpretation of the meaning.